src/hammer_config/config_src.py

```diff
@@ -336,6 +400,44 @@ def change_if_target(x: str) -> str:
                                            target_settings=crossref_targets,
                                            rename_target=crossref_rename)
 
+    def trycrossref_action(config_dict: dict, key: str, value: Any, params: MetaDirectiveParams) -> None:
+        """
+        Similar to crossref, but it is supposed to work whether you override
+        it with a concrete value downstream. For example. if originally a
+        key defaulted to a trycrossref of 'vlsi.foo.bar', and a later config
+        changed it to [1,2,3], this will be valid, and [1,2,3] is the final
+        value returned.
+        """
+        if type(value) == str:
```

Comment:
```suggestion
        if isinstance(value, str):
```

This applies generally for the `if type(...) == ...` pattern